From akpm@osdl.org Mon Oct 30 23:12:09 2006
Message-Id: <200610310712.k9V7C1G7020518@shell0.pdx.osdl.net>
From: Mariusz Kozlowski <m.kozlowski@tuxland.pl>
Subject: usb 'print_schedule_frame' defined but not used warning fix
Cc: m.kozlowski@tuxland.pl, greg@kroah.com
Date: Mon, 30 Oct 2006 23:12:01 -0800

From: Mariusz Kozlowski <m.kozlowski@tuxland.pl>

drivers/usb/host/ehci-sched.c:270: warning: 'print_schedule_frame' defined but not used

Signed-off-by: Mariusz Kozlowski <m.kozlowski@tuxland.pl>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ehci-sched.c |    2 ++
 1 file changed, 2 insertions(+)

--- gregkh-2.6.orig/drivers/usb/host/ehci-sched.c
+++ gregkh-2.6/drivers/usb/host/ehci-sched.c
@@ -265,6 +265,7 @@ static void print_budget (struct ehci_hc
 		print_budget_frame(ehci,i,insert,owner);
 }
 
+#if 0
 static void print_schedule_frame (char *pre,struct ehci_hcd *ehci, int frame,
 				  void *insert)
 {
@@ -333,6 +334,7 @@ static void print_schedule_frame (char *
 	}
 	printk("\n");
 }
+#endif
 
 /* find position of a specific entry in the periodic schedule (ie,
  * returns pointers such that we can update the predecessor's
